CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.18.0
+
+- feat: Registry target now adds key `createdAt` with current timestamp to json file
+
```

Comment:
craft will auto generate this based on your PR commit message so this shoudl be left out